src/components/interactive-builder/modals/question/question-form/common/sortable-tag/sortable-tag.scss

```diff
@@ -0,0 +1,58 @@
+@use '@carbon/colors';
+@use '@carbon/layout';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+$spacing-02: 8px;
+$spacing-04: 16px;
```

Comment:
```suggestion
```
You don't need to define these separately. Since these values are already imported by the line`@use '@carbon/layout';`, you should do `layout.$spacing-02` in the line that you need it.